src/Bundle/Resources/config/services/state/processor.php

```diff
@@ -0,0 +1,51 @@
+<?php
+
+/*
+ * This file is part of the Sylius package.
+ *
+ * (c) Sylius Sp. z o.o.
+ *
+ * For the full copyright and license information, please view the LICENSE
+ * file that was distributed with this source code.
+ */
+
+declare(strict_types=1);
+
+namespace Symfony\Component\DependencyInjection\Loader\Configurator;
+
+use Sylius\Resource\State\Processor\FlashProcessor;
+use Sylius\Resource\State\Processor\RespondProcessor;
+use Sylius\Resource\State\Processor\WriteProcessor;
+use Sylius\Resource\Symfony\Serializer\State\SerializeProcessor;
+
+return static function (ContainerConfigurator $container) {
+    $services = $container->services();
+    $parameters = $container->parameters();
+    $container->import('processor/write.php');
```

Comment:
Pretty sure this import is unneeded, it's covered by another import in a separate file